src/feature/hs/hs_cache.c

```diff
@@ -27,6 +27,15 @@
 static int cached_client_descriptor_has_expired(time_t now,
            const hs_cache_client_descriptor_t *cached_desc);
 
+/** Helper function: Return true iff the cache entry has a decrypted
+ * descriptor. */
```

Comment:
Please let's have some more function doc here. I know there is already in the struct, but let's be extra paranoid here to ensure that people that see this function learn about this behavior.